samples/subsys/modbus/tcp_server/src/main.c

```diff
@@ -26,6 +26,43 @@ static const struct gpio_dt_spec led_dev[] = {
 	GPIO_DT_SPEC_GET(DT_ALIAS(led2), gpios),
 };
 
+static int custom_read_count;
+
+static bool custom_handler(const int iface,
+			  const struct modbus_adu *rx_adu,
+			  struct modbus_adu *tx_adu,
+			  uint8_t *const excep_code,
+			  void *const user_data)
```

Comment:
Please add one more space after tab to indent properly.

Reply:
Had my git diffs showing the wrong tab sizes, which caused formatting to be a bit hard to judge. Should be fixed now.